test/tic_tac_toe/unit/game_presenter_test.exs

```diff
@@ -0,0 +1,9 @@
+defmodule GamePresenterTest do
+  use ExUnit.Case
+
+  test "renders a human-v-human new game" do
+    #wondering if I should have unit tests for this as the UI is partially tested
```

Comment:
Thanks for explaining that, it makes sense. I have added a cell presenter, which takes a cell and provides an abstraction to ask view-related questions about this cell. 

My understanding of a presenter was blurry, hopefully I have a grasp of it now. I have renamed the GamePresenter module to View_Manager, as this abstracts away from the controller specifics on how a view is rendered from raw data (currently a game struct)....Does this sound appropriate?

Thanks again for your feedback! :bowtie: